apps/right-to-rent-check/behaviours/tenancy-start-postcode-check.js

```diff
@@ -5,7 +5,12 @@ const pilotStartDate = new Date('2014/12/01').getTime();
 const pilotEndDate = new Date('2016/01/31').getTime();
 const _ = require('lodash');
 const inRange = date => date >= pilotStartDate && date <= pilotEndDate;
-const isPilot = postcode => _.some(pilotPostcodes, pilot => postcode.startsWith(pilot));
+const isPilotOrUndefined = postcode => {
+  if (postcode === undefined) {
+    return true;
+  }
```

Comment:
I could have put the logic below but it just made the code more messy

Reply:
does this mean that if the postcode has not been set, then the tenancy start date is valid?